draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1576,6 +1578,13 @@ UUIDs SHOULD be treated as opaque values and implementations SHOULD NOT examine
 the bits in a UUID. However,
 inspectors MAY refer to {{variant_field}} and {{version_field}} when required to determine UUID version and variant.
 
+As general guidance, we recommend not parsing UUID values unnecessarily,
+and instead treating them as opaquely as possible.  Although application-specific
+concerns could of course require some degree of introspection
+(e.g. to examine the variant, version or perhaps the timestamp of a UUID),
+the advice here is to avoid this or other parsing unless absolutely necessary.
+Applications typically tend to be simpler, more interoperable, and perform better,
+when this advice is followed.
```

Comment:
I think this helps give more guidance on the opacity concern.  My rationale here is that people writing UUID libraries and such will have to make a decision about how much parsing capability to include, and I don't want people to feel compelled to provide a bunch of detailed parsing, when one of the whole drives of these new UUIDs is that they are more useful as opaque values.  So with this wording I think it's more clear and will ease the implementation burden.